legal-auto.py

```diff
@@ -136,7 +138,7 @@ def check_source_submission(self, src_project, src_package, src_rev, target_proj
                 # reopen
                 return None
             if state == 'new' and self.valid_for_opensuse(target_project, report):
-                self.message = 'The legal review is accepted preliminary. The package may require actions later on.'
+                self.message = 'Legal review pending, opensuse-review-team is responsible for license compliance'
```

Comment:
That statement is untrue and was never agreed in this form. The review team cannon be held responsible (as half of the team would not even have access to legaldb)

The plan was that the things are accepted preliminary - but stay open on legaldb to be reviewed and taken action whenever the legal team has the capacity to review the stuff. Nothing more, nothing less.

Reply:
That might be a problem, because it's certainly not me either. Who from legal has agreed to take responsibility for preliminary accepted requests? Looking through the data, it appears since 2022 only 14 such reports have actually been reviewed after being auto-accepted.